src/ontology/uberon-edit.obo

```diff
@@ -64,59 +64,59 @@ import: http://purl.obolibrary.org/obo/uberon/imports/pato_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/pr_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/ro_import.owl
 ontology: uberon/core
-property_value: dc-contributor "Alan Ruttenberg - https://orcid.org/0000-0002-1604-3078" xsd:string
```

Comment:
`dc-contributor` is not a well-formed CURIE. should be `dc:contributor`

Reply:
dont worry - this is just an ideosyncracy of OBO format. It will come out correctly in RDF/OWL